packages/web-pkg/tests/unit/components/sidebar/Spaces/Details/SpaceDetailsMultiple.spec.ts

```diff
@@ -0,0 +1,68 @@
+import SpaceDetailsMultiple from 'web-pkg/src/components/sideBar/Spaces/Details/SpaceDetailsMultiple.vue'
+import { spaceRoleManager, ShareTypes } from 'web-client/src/helpers/share'
+import {
+  createStore,
+  defaultPlugins,
+  shallowMount,
+  defaultStoreMockOptions
+} from 'web-test-helpers'
+
+const spaceMock = {
+  type: 'space',
+  name: ' space',
+  id: '1',
+  mdate: 'Wed, 21 Oct 2015 07:28:00 GMT',
+  spaceQuota: {
+    used: 100,
+    total: 1000,
+    remaining: 900
+  }
+}
+
+const spaceShare = {
+  id: '1',
+  shareType: ShareTypes.space.value,
+  collaborator: {
+    onPremisesSamAccountName: 'Alice',
+    displayName: 'alice'
+  },
+  role: {
+    name: spaceRoleManager.name
+  }
+}
```

Comment:
This file looks like a lot of copy-paste, which is actually not needed here. You can probably remove `spaceShare` and all the store-related stuff in `createWrapper` (except for `defaultStoreMockOptions` maybe).

```suggestion
```